### ISSUE TITLE
Allow users to specify the start page of a custom account-console theme

### ISSUE THREAD

**Issue by thomasdarimont** [keycloak/keycloak#35639]
### Description

As a developer I want to specify the page users see when they access `/realm/${REALM}/account-console/`.

### Motivation

Currently the profile page is shown by default, however in some environments we only want users to change their password or register a 2FA mechanism and without the ability to update the profile.

Currently the personal info page is shown by default and cannot (?) be hidden by default.

### Details

Also when users build custom account-consoles they might want to show a different default page instead of the personal info page.

It would be convenient if the default page could be configured via code / theme properties.

**Comment by edewit** [comment]
@thomasdarimont you can already do this by making the `path: ""` in the content.json

**Comment by thomasdarimont** [comment]
Thanks @edewit !

Do you have a brief example for this?
In my case I only want to show the password / otp page in the account console.

**Comment by edewit** [comment]
Just override the content.json like in this example: https://github.com/keycloak/keycloak-quickstarts/blob/main/extension/extend-account-console/src/main/resources/theme/keycloak-man/account/resources/content.json

**Comment by thomasdarimont** [comment]
Hello @edewit ,

I just tried this with KC 26.0.7 and the following `content.json`:
```json
[
  {
    "label": "accountSecurity",
    "children": [
      { "label": "signingIn", "path": "account-security/signing-in" }
    ],
    "path": "/"
  }
]
```

However, when I access $ISSUER/account/ I still see the profile page and not the accountSecurity page.

Is there anything I'm missing?

**Comment by edewit** [comment]
No, you are right that doesn't work, I'll create a PR to fix this

**Comment by peschee** [comment]
@edewit Has this been released in Keycloak `26.0.8`? I can't find this Issue number on the releases page.
